eng/common/TestResources/New-TestResources.ps1

```diff
@@ -56,7 +56,6 @@ param (
     [string] $Environment = 'AzureCloud',
 
     [Parameter()]
-    [ValidateNotNullOrEmpty()]
```

Comment:
This gets used correctly when empty then? I expect so, just making sure that was tested.

I recommend, if you didn't already, trying this in a repo where this gets used and porting over after trying it.

Reply:
Here's an example of porting KV tests with the code that is in place today: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=350004&view=logs&j=ad07b744-5a1a-5d54-7693-fcb02e666633&t=d2c457f2-9cbc-5cd7-578d-8751a7b8b565&l=100

Here's the KV tests running when the change is (temporarily) in place in the eng/common/TestResources: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=350004&view=logs&j=ad07b744-5a1a-5d54-7693-fcb02e666633&t=d2c457f2-9cbc-5cd7-578d-8751a7b8b565&l=100